### ISSUE TITLE
Legend should indicate dash/fill patterns in addition to color

### ISSUE THREAD

**Issue by gazialankus** [google/charts#280]
Right now, when I use a legend with dashed line charts, the legend only has the color and not the dashes. It should also display the dashes. This is important for people who are color blind and for printing out the chart in b/w. I'm using this library with Flutter Web and users will actually need to print out the charts.

Below I have a screenshot:

![image](https://user-images.githubusercontent.com/412997/60097952-2cbf5480-975d-11e9-8218-a2f053b3b788.png)

**Comment by KingLudwig94** [comment]
Hi, how have you managed to use this package with Flutter Web? On pub.dev it states it is for app only. I'm trying but it gives me an error about Type not being assigned to Widget

**Comment by gazialankus** [comment]
Yes, you can use it from here for Flutter Web: https://github.com/gazialankus/charts_flutter_web

**Comment by KingLudwig94** [comment]
Thanks a lot!
